src/scope-selector-parser.pegjs

```diff
@@ -23,6 +23,10 @@ path
     return new matchers.PathMatcher(prefix, first, others);
   }
 
+  / prefix:([LRB]":")? parent:(scope _)+ ">" descendants:(_ scope)+ {
```

Comment:
1. This isn't valid vs spec - descendant selector can be used ad infinitum.
2. No need to put in a second rule. Can just make the `PathMatcher.matches` loop a bit more complex.

Reply:
Resolved in 15d1ce4b1d7ee46eb39c58987750b14988f3421b